Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -715,6 +715,326 @@ public void When_a_deeply_nested_property_of_a_collection_with_an_invalid_value_
             act.Should().NotThrow();
         }
 
+        [Fact]
```

Comment:
🔧 We've [started to wrap](https://github.com/fluentassertions/fluentassertions/discussions/1551) related tests in a nested class. This also helps keep the test names shorter and functional. In this case, wrapping all tests dealing with `Excluding` and `ThenExcluding` would make total sense to me.
❌ Please avoid using the literal name of a method in test names. This makes the test name very difficult to read. Also applies to the other tests.

Reply:
✅ Extracted `ThenExcluding` tests to a nested class